eagerpy/tensor/tensorflow.py

```diff
@@ -176,6 +176,12 @@ def argsort(self: TensorType, axis: Optional[int] = -1) -> TensorType:
     def sort(self: TensorType, axis: Optional[int] = -1) -> TensorType:
         return type(self)(tf.sort(self.raw, axis=axis))
 
+    def topk(
+        self: TensorType, k: int, sorted: bool = True
+    ) -> Tuple[TensorType, TensorType]:
+        pair = tf.math.top_k(self.raw, k, sorted=sorted)
```

Comment:
You can use tuple unpacking instead: `values, indices = tf.math…`.
It's more descriptive and avoids tuple indexing in the next line.